api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -170,8 +178,15 @@
     public ResponseEntity<Resource> downloadFile(
             @ApiParam(value = "A job identifier", required = true) @PathVariable @NotBlank String jobId,
             @ApiParam(value = "A file name", required = true) @PathVariable @NotBlank String filename) throws IOException {
-        HttpHeaders responseHeaders = new HttpHeaders();
-        responseHeaders.add("X-Progress", "0%");
-        return new ResponseEntity<>(null, responseHeaders, HttpStatus.ACCEPTED);
+        Path file = Paths.get(fileDownloadPath + filename);
```

Comment:
Getting a job's output file is more involved. We will need a JobService method to encapsulate that logic: JobService.getJobOutput(jobID, fileName) returns `Resource`. Explanation to follow.